e2e/package.json

```diff
@@ -13,7 +13,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "cypress": "^9.4.1",
+    "cypress": "9.5.1",
```

Comment:
At the moment, this is actually a downgrade. `^9.x.x` will always pull the latest major 9 version because the e2e folder does not have a package-lock. See also #1102.

The exact version pinning here is needed to extract the version from the package.json to build the correct docker image.